src/init.cpp

```diff
@@ -250,11 +250,6 @@ void PrepareShutdown()
         DumpMempool();
     }
 
-    // After everything has been shut down, but before things get flushed, stop the
-    // CScheduler/checkqueue threadGroup
-    threadGroup.interrupt_all();
-    threadGroup.join_all();
```

Comment:
why moving these to the bottom?
Shouldn't we stop the threadGroup before flushing things?

Reply:
Nevermind, you've fixed it later in bdcadcab5136e29fb3237c403e20ac672d074ffb (it could be squashed here though).

Reply:
yeah, i forgot to squash it. Here i discovered the issue solved in #2191.